README.md

```diff
@@ -25,7 +25,7 @@ If your issue is not related to the Office Scripts documentation, please post it
 
 - To ask a question about designing Office Scripts or the Office.js API that runs Office Scripts, post your question to Stack Overflow and tag it with the "office-scripts" tag (https://stackoverflow.com/questions/tagged/office-scripts).
 - To report an issue with the Office.js API, create the issue in the [OfficeDev/office-js repository](https://github.com/OfficeDev/office-js), which members of the product team monitor for customer-reported issues.
-- To submit a feature request for Office Scripts, post your idea to [Microsoft Q&A](https://docs.microsoft.com/answers/products/m365), or if the feature request already exists there, add your vote for it. Be sure to use the "Office-Scripts-Dev" tag when asking for your feature.
+- To submit a feature request for Office Scripts, post your idea the Excel Tech Community (https://techcommunity.microsoft.com/t5/excel/bd-p/ExcelGeneral), or if the feature request already exists there, add your vote for it. Be sure to use the "Office Scripts" tag when asking for your feature.
```

Comment:
@AlexJerabek This doesn't seem quite right. The ExcelGeneral community only has discussion and blogs. They do not have a "feature request" section (called "Ideas" in the M365 dev platform space) so there is no way to vote. Also, I see that just above in line 26, the topic points people to Stack. 
1. We want to suggest people ask Office Script questions on Microsoft Q+A
2. We want "ideasFeature requests/suggestions that did go to UserVoice to go to TC Ideas so they can get voted on. But, the only Ideas section we have is in Microsoft 365 develop platform. I will suggest we add a tag there, for feature requests?

Reply:
Oh, I misunderstood that "Ideas section" point in our related thread.